src/Testing/MakesGraphQLRequests.php

```diff
@@ -227,4 +230,28 @@ protected function rethrowGraphQLErrors(): void
         $config = Container::getInstance()->make(ConfigRepository::class);
         $config->set('lighthouse.error_handlers', [RethrowingErrorHandler::class]);
     }
+
+    protected function setupSubscriptionEnvironment(): void
+    {
+        config()->set('lighthouse.subscriptions.queue_broadcasts', false);
+        config()->set('lighthouse.subscriptions.storage', 'array');
+        config()->set('lighthouse.subscriptions.storage_ttl', null);
+
+        // binding an instance to the container so it can be spied on
+        app()->bind(LogBroadcaster::class, function () {
+            return new LogBroadcaster(config('lighthouse.subscriptions.broadcasters.log'));
+        });
+
+        assert($this->app instanceof Application);
+        $broadcastManager = $this->app->make(BroadcastManager::class);
+        assert($broadcastManager instanceof BroadcastManager);
+
+        // adding a custom driver which is a spied version of log driver
+        $broadcastManager->extend('mock', function ($app, $config) {
```

Comment:
types? are the args even necessary?

Reply:
No you're right, they are not anymore at first I was doing something with them but thought of a better way